### ISSUE TITLE
information about binding context of compose is hard to find

### ISSUE THREAD

**Issue by dkent600** [aurelia/documentation#287]
I am not finding an description of what is the binding context of a view when composed using the "compose" element.

Seems like I've seen that somewhere, but I think it should be a lot easier to find.

I would suggested putting that information here:  http://aurelia.io/hub.html#/doc/article/aurelia/templating/latest/templating-basics/3

**Comment by EisenbergEffect** [comment]
Good point. I've flagged this to add to our docs. In the mean time, I can answer your question. What is it specifically that you are wondering about? This can also help us to determine what info to add to the article.

**Comment by dkent600** [comment]
:-) Thanks @EisenbergEffect .

First, this is in reference to the [larger question I just posed on stackoverflow](http://stackoverflow.com/questions/39834025/is-it-necessary-to-define-a-template-in-a-separate-file).  I'd love to also have an answer to that, and if it is in the affirmative, it would be great to see that in the docs too.

I would like to know whether one has access *by default* in the composed template to the container context?

Otherwise, my guess would be that one could pass in the container context, or whatever one wants, view the `model` atttribute of `compose`.

**Comment by Alexander-Taran** [comment]
seems covered in 
http://aurelia.io/docs/templating/dynamic-ui-composition#basic-composition

**Comment by dkent600** [comment]
Yeah, though the question I posed is not answered:  Does the composed view have any access, by default (specifying neither model nor view-model), to the containing context?
